docs-src/app-dev-context/environment-variables.md

```diff
@@ -8,7 +8,7 @@ tags:
 ---
 
 Environment variables can be provided in the normal way for our language to our Client, Worker, and Activity code.
-They can't be used normally with Workflow code, as that would be [nondeterministic](workflows#intrinsic-non-deterministic-logic) (if the environment variables changed between Workflow replays, the code that used them would behave differently).
+They can't be used normally with Workflow code, as that would be [nondeterministic](/workflows#intrinsic-non-deterministic-logic) (if the environment variables changed between Workflow replays, the code that used them would behave differently).
```

Comment:
[nondeterministic](/concepts/what-is-a-workflow-definition#intrinsic-non-deterministic-logic)